payment/src/main/java/com/kyriba/conference/payment/api/PaymentMethodController.java

```diff
@@ -0,0 +1,68 @@
+package com.kyriba.conference.payment.api;
+
+import com.kyriba.conference.payment.domain.PaymentMethodType;
+import com.kyriba.conference.payment.domain.dto.PaymentMethodDto;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.kyriba.conference.payment.domain.PaymentMethodType.*;
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+/**
+ * @author Garik Lizura
+ */
+@RestController
+@AllArgsConstructor
+@RequestMapping("/v1/paymentMethod")
+public class PaymentMethodController {
```

Comment:
The same set of comments as for another controller.

Reply:
done